src/item/StringToItemParser.php

```diff
@@ -869,7 +869,7 @@ private static function registerBlocks(self $result) : void{
 		$result->registerBlock("pink_tulip", fn() => Blocks::PINK_TULIP());
 		$result->registerBlock("piglin_head", fn() => Blocks::MOB_HEAD()->setMobHeadType(MobHeadType::PIGLIN));
 		$result->registerBlock("pitcher_plant", fn() => Blocks::PITCHER_PLANT());
-		$result->registerBlock("pitcher_pod", fn() => Blocks::PITCHER_CROP());
+		$result->registerBlock("pitcher_crop", fn() => Blocks::PITCHER_CROP());
```

Comment:
This isn't normally available via /give

Reply:
Blocks like wheat_block, potato_block, carrot_block are not normally available via /give too but they are registered in STIP

Reply:
That's because they were available in the legacy Item::fromString(), so they have to be kept for BC reasons. New items should not be doing this.